ports/javascript/js_module.c

```diff
@@ -0,0 +1,39 @@
+#include "py/nlr.h"
+#include "py/obj.h"
+#include "py/runtime.h"
+#include "py/binary.h"
+
+#include <stdio.h>
+
+// forward declare emscripten stuff
+void emscripten_run_script(const char *);
+int emscripten_run_script_int(const char *);
+
+STATIC mp_obj_t js_eval(mp_obj_t js_string) {
+    emscripten_run_script(mp_obj_str_get_str(js_string));
+    return mp_const_none;
+}
+
+STATIC mp_obj_t js_evali(mp_obj_t js_string) {
```

Comment:
Are there any other return types that could be supported, like float?  They don't have to be added now, but good to think how they would be consistently named (eg evalf) if they were added in the future.